browser/react/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// import { Provider } from 'react-redux';
+// import store from '../redux/store';
+import { Router, Route, browserHistory, IndexRedirect } from 'react-router';
+import App from './components/App';
+import 'aframe';
```

Comment:
Need sockets. Doesn't seem to need aframe, but I say that with reservation (1st time I tried it without, didn't work. Following two times - without having changed anything else - it was fine). We'll take it out for now and can add it back in later if it's giving us problems.